test/Tests/src/Runtime/Stack_Traces_Spec.enso

```diff
@@ -0,0 +1,20 @@
+from Standard.Base import all
+import Standard.Test
+
+type My_Type
+
+bar = Runtime.get_stack_trace
+baz = here.bar
+Number.foo = here.baz
+foo x = x.foo
+My_Type.foo = here.foo 123
+
+spec = Test.group "Stack traces" <|
+    Test.specify "should capture traces correctly" <|
+        modname = Meta.Constructor (Meta.meta here . constructor) . name
+        stack = My_Type.foo
+        names = [modname + ".bar", modname + ".baz", "Number.foo", modname + ".foo", "My_Type.foo"]
```

Comment:
I was thinking if the names should be qualified.
Does it capture the name, as it was resolved at the `get_stack_trace` call point? Or are the module names simply omitted?

Reply:
Qualified names are too noisy most of the time. But there's nothing stopping us from exposing both if we really wanted to (with the caveat that they will be missing in polyglot calls).